packages/core/core/index.d.ts

```diff
@@ -1,5 +1,32 @@
 declare module "embark-core" {
   export class IPC {
-    constructor(options: {ipcRole: string});
+    constructor(options: { ipcRole: string });
+  }
+
+  type CommandCallback = (
+    opt1?: any,
+    opt2?: any,
+    opt3?: any,
+    opt4?: any,
+    opt5?: any,
+    opt6?: any,
+    opt7?: any,
+    opt8?: any,
+    opt9?: any,
+    opt10?: any,
+    opt11?: any,
+    opt12?: any,
+  ) => any;
```

Comment:
I wholeheartedly agree with this, I simply moved the code from another place and didn't want to also change it, as I thought it might cloud the purpose of the PR.